tpot/export_utils.py

```diff
@@ -88,6 +90,11 @@ def export_pipeline(exported_pipeline, operators, pset):
     train_test_split(features, tpot_data['class'], random_state=42)
 """
 
+    if pscore is not None:
+        pipeline_text += '\n# Score on the training set was:{}\n'.format(pscore)
+    else:
+        pipeline_text += '\n'
```

Comment:
Please refactor to:

```Python
    if pscore is not None:
         pipeline_text += '\n# Score on the training set was: {}'.format(pscore)
    pipeline_text += '\n'
```

Please also rename `pscore` to `pipeline_score` for a more descriptive variable name.